DogServiceExample/src/main/java/rest/DogService.java

```diff
@@ -1,141 +0,0 @@
-package rest;
-
-import Json.JsonHelper;
-import app.Dog;
-
-import com.google.gson.JsonElement;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import org.json.simple.parser.ParseException;
-
-import javax.ws.rs.*;
-import javax.ws.rs.core.Response;
-import java.util.HashMap;
-import java.util.Map;
-
-
-@Path("/")
-public class DogService {
-
-    private static Integer id = 0;
-    private static Map<Integer, Dog> dogs = new HashMap<Integer, Dog>();
-
-    static {
-        Dog dog0 = new Dog();
-        dog0.setId(id);
-        id++;
-        dog0.setName("Loly");
-        dog0.setDateOfBirth("08.11.2013");
-        dog0.setHeight(50);
-        dog0.setWeight(7);
-        dogs.put(dog0.getId(), dog0);
-
-		Dog dog1 = new Dog();
-		dog1.setId(id);
-		id++;
-		dog1.setName("Pony");
-		dog1.setDateOfBirth("05.10.2011");
-		dog1.setHeight(50);
-		dog1.setWeight(7);
-		dogs.put(dog1.getId(), dog1);
-
-//		Dog dog2 = new Dog();
-//		dog2.setId(String.valueOf(id));
-//		id++;
-//		dog2.setName("Rainbow");
-//		dog2.setDateOfBirth("08.12.2015");
-//		dog2.setHeight(61);
-//		dog2.setWeight(12);
-//		dogs.put(dog2.getId(), dog2);
-    }
-
-    @GET
-    @Path("dogs")
-    @Produces("application/json")
-    public Response printMessage() {
-        return Response.status(200).entity(JsonHelper.mapToJson(dogs)).build();
-    }
-
-
-	@GET
-	@Path("dog/{id}")
-	@Produces("application/json")
-	public Response printMessage(@PathParam("id") String id) {
-		JsonElement jelement = new JsonParser().parse(JsonHelper.mapToJson(dogs));
-		JsonObject jobject = jelement.getAsJsonObject();
-		JsonObject result = jobject.getAsJsonObject(id);
```

Comment:
This code makes no sense. You are asked to get one dog by id, but you serialize all the dogs, then get one.

Also why is the method named `printMessage`?